pull_traces.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# The following command pulls all files ending in .perfetto-trace from the directory
+# hierarchy starting at the root /storage/emulated/0/Android.
+adb shell find /storage/emulated/0/Android/ -name "*.perfetto-trace" \
```

Comment:
do you find this script to be useful as is, since benchmark will happily keep creating new trace files, with new names?

was curious about that, when reviewing docs

Reply:
I thought it may be useful for folks checking the samples and directly pulling the things.
But I think if AS or gradle copies the files properly (the comment above), it may not be needed 🤔